clusters/build-clusters/common/hack/generate-bw-items.sh

```diff
@@ -11,8 +11,13 @@ set -o pipefail
 WORKDIR="$(mktemp -d)"
 readonly WORKDIR
 
-CONTEXT="build02"
-readonly CONTEXT
+CONTEXT="$(oc config current-context)"
```

Comment:
Please explicitly state the contexts we expect

Reply:
Updated